chore(wallet)_: remove unused fields from the token type

saledjenic opened the pull request:
Before this change updatedAt was pointing to the when the last list from the stores was updated. That's sorted out.

`LastUpdateTimestamp` field added to the `List` type. A client needs to display the time of the last update.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (46)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/1/) | 2025-02-27 14:06:40 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250227-140329-0f22d5-pr6375.zip) |
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/1/) | 2025-02-27 14:06:40 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250227-140327-0f22d5-pr6375.aar) |
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/1/) | 2025-02-27 14:08:14 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250227-140331-0f22d5-pr6375.zip) |
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/1/) | 2025-02-27 14:08:45 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250227-140332-0f22d5-pr6375.zip) |
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/1/) | 2025-02-27 14:09:28 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250227-140335-0f22d5-pr6375.zip) |
| :heavy_check_mark: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/1/) | 2025-02-27 14:09:44 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250227-140330-0f22d5-pr6375.zip) |
| :heavy_multiplication_x: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/1/) | 2025-02-27 14:17:39 | ~14 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/1//consoleText) |
| :heavy_multiplication_x: | 0f22d58a | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/1/) | 2025-02-27 14:36:06 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/2/) | 2025-03-03 07:35:02 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250303-073202-f472a7-pr6375.aar) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/2/) | 2025-03-03 07:35:12 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250303-073202-f472a7-pr6375.zip) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/2/) | 2025-03-03 07:36:44 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-073205-f472a7-pr6375.zip) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/2/) | 2025-03-03 07:36:52 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250303-073210-f472a7-pr6375.zip) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/2/) | 2025-03-03 07:37:24 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-073206-f472a7-pr6375.zip) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/2/) | 2025-03-03 07:38:16 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250303-073204-f472a7-pr6375.zip) |
| :heavy_multiplication_x: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/2/) | 2025-03-03 07:45:29 | ~13 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/2//consoleText) |
| :heavy_check_mark: | f472a7ae | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/2/) | 2025-03-03 08:04:00 | ~31 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/3/) | 2025-03-03 10:11:24 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250303-100807-4b44c2-pr6375.aar) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/3/) | 2025-03-03 10:11:36 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250303-100808-4b44c2-pr6375.zip) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/3/) | 2025-03-03 10:12:05 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-100810-4b44c2-pr6375.zip) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/3/) | 2025-03-03 10:12:16 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250303-100815-4b44c2-pr6375.zip) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/3/) | 2025-03-03 10:13:27 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-100811-4b44c2-pr6375.zip) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/3/) | 2025-03-03 10:14:17 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250303-100809-4b44c2-pr6375.zip) |
| :heavy_multiplication_x: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/3/) | 2025-03-03 10:22:15 | ~14 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/3//consoleText) |
| :heavy_check_mark: | 4b44c2a4 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/3/) | 2025-03-03 10:37:46 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/4/) | 2025-03-03 12:24:36 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250303-122125-8c6cda-pr6375.aar) |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/4/) | 2025-03-03 12:25:24 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-122129-8c6cda-pr6375.zip) |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/4/) | 2025-03-03 12:25:42 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250303-122126-8c6cda-pr6375.zip) |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/4/) | 2025-03-03 12:25:51 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250303-122133-8c6cda-pr6375.zip) |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/4/) | 2025-03-03 12:27:27 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-122129-8c6cda-pr6375.zip) |
| :heavy_check_mark: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/4/) | 2025-03-03 12:28:54 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250303-122127-8c6cda-pr6375.zip) |
| :heavy_multiplication_x: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/4/) | 2025-03-03 12:33:37 | ~12 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/4//consoleText) |
| :heavy_multiplication_x: | 8c6cdad8 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/4/) | 2025-03-03 12:51:24 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/5/) | 2025-03-03 15:53:19 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250303-155016-b28222-pr6375.aar) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/5/) | 2025-03-03 15:54:15 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250303-155017-b28222-pr6375.zip) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/5/) | 2025-03-03 15:54:42 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250303-155025-b28222-pr6375.zip) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/5/) | 2025-03-03 15:55:55 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250303-155019-b28222-pr6375.zip) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/5/) | 2025-03-03 15:55:56 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-155020-b28222-pr6375.zip) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/5/) | 2025-03-03 15:56:22 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-155021-b28222-pr6375.zip) |
| :heavy_multiplication_x: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/5/) | 2025-03-03 16:12:52 | ~22 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/5//consoleText) |
| :heavy_check_mark: | b2822261 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/5/) | 2025-03-03 16:22:43 | ~32 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/5//consoleText) |
| | | | | | | |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/6/) | 2025-03-03 15:55:46 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/6/consoleText) |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/6/) | 2025-03-03 15:56:46 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/6/consoleText) |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/6/) | 2025-03-03 15:56:55 | ~2 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/6/consoleText) |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/6/) | 2025-03-03 15:58:24 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/6/consoleText) |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/6/) | 2025-03-03 15:58:39 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/6/consoleText) |
| :x: | b1509b8f | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/6/) | 2025-03-03 16:00:02 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/6/consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/7/) | 2025-03-03 16:15:36 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/7/consoleText) |
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/7/) | 2025-03-03 16:16:07 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/7/consoleText) |
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/7/) | 2025-03-03 16:16:13 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/7/consoleText) |
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/7/) | 2025-03-03 16:16:15 | ~2 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/7/consoleText) |
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/7/) | 2025-03-03 16:16:16 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/7/consoleText) |
| :x: | b0531917 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/7/) | 2025-03-03 16:17:59 | ~4 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/7/consoleText) |
| :heavy_multiplication_x: | b0531917 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/6/) | 2025-03-03 16:26:18 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6375/8/) | 2025-03-03 16:55:41 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250303-165239-bde82a-pr6375.zip) |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6375/8/) | 2025-03-03 16:55:53 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250303-165238-bde82a-pr6375.aar) |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6375/8/) | 2025-03-03 16:57:00 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250303-165247-bde82a-pr6375.zip) |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6375/8/) | 2025-03-03 16:57:20 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-165243-bde82a-pr6375.zip) |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6375/8/) | 2025-03-03 16:57:56 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250303-165243-bde82a-pr6375.zip) |
| :heavy_check_mark: | bde82ae1 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6375/8/) | 2025-03-03 16:57:59 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250303-165240-bde82a-pr6375.zip) |
| :heavy_check_mark: | bde82ae1 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/7/) | 2025-03-03 17:10:37 | ~17 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6375/7//consoleText) |
| :heavy_check_mark: | bde82ae1 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/7/) | 2025-03-03 17:23:15 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6375/7//consoleText) |

dlipicar commented on services/wallet/reader.go:
```diff
@@ -396,34 +381,29 @@ func (r *Reader) getBalance1DayAgo(balance *token.ChainBalance, dayAgoTimestamp
-					BalancesPerChain: balancesPerChain,
-				}
+		for symbol, tokens := range getTokenBySymbols(allTokens) {
+			balancesPerChain := r.createBalancePerChainPerSymbol(address, balances, tokens, cachedTokens, connectedPerChain, dayAgoTimestamp)
```
the goal behind splitting verified and unverified tokens was so that, for example, balances of some random community token with symbol DAI would not be grouped together with (or even overwrite) the balances for the real DAI. We mark tokens coming from a token list as "verified" and expect symbols to be unique among verified tokens. It's a workaround since we're using the symbols as a key to identify a token across multiple chains, we cannot remove the workaround until we identify our tokens somehow else.

dlipicar replied:
Not sure if some change was done in the meantime to account for that, making the workaround unnecessary. Please test the condition I speak about (i.e. create a community token with symbol DAI and check that the balances are not mixed up)

dlipicar replied:
we should probably have a unit test covering that scenario

saledjenic replied:
> It's a workaround since we're using the symbols as a key to identify a token across multiple chains

Yes, that was before (that's even now), but this PR, as well as the base PR of this one, are just small steps towards the final idea where we will make tokens unique by chain + address combination cause that's the only unique thing.

> We mark tokens coming from a token list as "verified" and expect symbols to be unique among verified tokens.

I wonder how we handle (in the current code, from develop) if there is more than a single community and each has the same symbol for their token? I guess in that case with this approach, even Verified flag doesn't help, no?

I don't mind reverting changes for the Verified flag, keeping the reset, and getting back to it in one of the follow-up PRs, sound good?

saledjenic replied:
> until we identify our tokens somehow else

Can't we say that tokens with `CommunityData != nil` are actually custom tokens, not verified?

saledjenic committed:
chore(wallet)_: remove unused field from the token type

Removed fields:
- `TokenListID`

saledjenic committed:
chore(wallet)_: associating the token list timestamp with the corresponding field in the response

Before this change updatedAt was pointing to the when the last list from the stores was updated. That's sorted out.

`LastUpdateTimestamp` field added to the `List` type. A client needs to display the time of the last update.

saledjenic commented:
I've just reverted the `Verified` field. Please have a look at the changes now. @all

friofry reviewed (approved):
Looks good to me. I've grepped `TokenListID` and it doesn't seem to be used anywhere.
